test/signalrclient-e2e-tests/connection_tests.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "stdafx.h"
+#include <string>
+#include "cpprest\details\basic_types.h"
+#include "cpprest\json.h" 
+#include "connection.h"
+#include "hub_connection.h"
+#include "url.h"
+
+TEST(connection_tests, connection_status_start_stop)
+{
+    auto conn = std::make_shared<signalr::connection>(url + U("raw-connection"));
+    auto test_event = std::make_shared<pplx::event>();
```

Comment:
remove `test_event` - you are now blocking and so the code is running synchronously and this event does not add any value.